src/main/java/edu/ie3/datamodel/io/csv/BufferedCsvWriter.java

```diff
@@ -110,18 +137,26 @@ public Writer append(char c) {
   @Override
   public boolean equals(Object o) {
```

Comment:
Why this change in `equals()`?

Reply:
Bc the outline of the classes attributes has changed (unpacking the information from former `CsvFileDefinition`, that has been the only attribute).